src/qumin/representations/patterns.py

```diff
@@ -953,67 +960,86 @@ def to_md(self, md, pair, folder, kind):
                     pair_table=table.to_markdown(index=False, headers=["lexeme", a, b])
                 ))
 
-    def to_csv(self, md, folder, kind):
+        md.register_file(f"{rel_path}{name}.md",
+                         name="human_" + name,
+                         custom=dict(cells=pair,
+                                     kind="human_readable",
+                                     algorithm=kind),
+                         description=f"Human-readable patterns between cells '{a}' and '{b}', "
+                                     f"with the '{kind}' algorithm.")
+
+    def to_csv(self, md, kind, path="patterns/machine_readable/"):
         """Export a Patterns DataFrame to csv."""
 
+        # Path settings
+        abs_path = md.get_path(path)
+        log.info("Writing patterns (importable by other scripts) to %s", abs_path)
+
         # Patterns map
         pattern_list = set()
         for pair in self:
             patterns = self[pair]['pattern'].unique()
             pattern_list.update([repr(pat) for pat in patterns])
         pattern_map = {pat: n for n, pat in enumerate(pattern_list)}
-        filename = md.register_file("patterns_map.csv")
         s = pd.Series({n: pat for pat, n in pattern_map.items()}, name="patterns")
-        s.to_csv(filename)
+        s.to_csv(abs_path + "/patterns_map.csv")
+        md.register_file(path + "patterns_map.csv")
 
         # One csv per pair of cells
         for (a, b) in self.keys():
-            filename = md.register_file(f"{kind}_{a}-{b}.csv", folder=folder)
+            name = f"pat_{kind}_{kind}_{a}-{b}.csv"
             export = self[(a,b)].copy()
-            if type(export.pattern.iloc[0]) != str:
+
+            if not isinstance(export.pattern.iloc[0], str):
                 export.pattern = export.pattern.map(repr)
 
             # Replace forms by ids
             export[['form_x', 'form_y']] = \
                 export[['form_x', 'form_y']].map(lambda x: x.id)
-
             # Replace patterns by ids
             export.pattern = export.pattern.map(pattern_map)
-            export.drop(["lexeme"], axis=1).to_csv(filename, sep=",", index=False)
-
-    def from_file(self, folder, *args, force=False, cells=None, **kwargs):
+            export.drop(["lexeme"], axis=1).to_csv(abs_path + "/" + name, sep=",", index=False)
```

Comment:
Using a literal slash here instead of using Path() for path concatenation is not compatible on all OSes, I think.